pkg/monitoring/vms/prometheus/prometheus_test.go

```diff
@@ -104,6 +104,27 @@ var _ = Describe("Prometheus", func() {
 			Expect(result.Desc().String()).To(ContainSubstring("kubevirt_vmi_memory_available_bytes"))
 		})
 
+		It("should send unused memory", func() {
+			ch := make(chan prometheus.Metric, 1)
+			defer close(ch)
+
+			ps := prometheusScraper{ch: ch}
+
+			vmStats := &stats.DomainStats{
+				Cpu: &stats.DomainStatsCPU{},
+				Memory: &stats.DomainStatsMemory{
+					UnusedSet: true,
+					Unused:    2048,
+				},
+			}
+			vmi := k6tv1.VirtualMachineInstance{}
+			ps.Report("test", &vmi, vmStats)
+
+			result := <-ch
+			Expect(result).ToNot(BeNil())
+			Expect(result.Desc().String()).To(ContainSubstring("kubevirt_vmi_memory_unused_bytes"))
```

Comment:
can we also get an expects case to ensure the conversion of kib to bytes is accurate

Reply:
The result is of type `prometheus.Metric` and it does not have any method that returns its value, or at least I couldn't find anything about it yet :thinking: 

I do see a lot of tests at [tests/infra_test.go](https://github.com/kubevirt/kubevirt/blob/master/tests/infra_test.go) that retrieves metrics values, but it might be difficult to do unit tests with that approach

Reply:
maybe I could create a new function for the conversion and a test for it as well, but I think that it would be just way too simple to need a test for it... WDYT?

Reply:
just something that validates the conversion. whatever path needs to be taken to do that. We had some issues with memory conversions recently, which made it apparent this is a pretty bad thing for us to break.

Reply:
I think you can write it to dto.Metric and from that verify the value?

Reply:
Unit tests added for the new metric and for others memory metrics as well
A lot easier than I thought, thanks to @xpivarc suggestion